src/Components/Web/src/Forms/InputDate.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.AspNetCore.Components.Forms;
 
 /// <summary>
 /// An input component for editing date values.
-/// Supported types are <see cref="DateTime"/> and <see cref="DateTimeOffset"/>.
+/// Supported types are <see cref="DateTime"/>, <see cref="DateOnly"/> and <see cref="DateTimeOffset"/>.
```

Comment:
TimeOnly?

Reply:
Indeed, the code in this file seems to indicate that's also supported, but I'm not an expert in this area.

Reply:
I'm not either but I just looked at the file 😀 let's put both

Reply:
good catch, seems about right. Merci